config/redis.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const config = require('config');
+
+const redisConfig = config.get('redis');
+
+const queuePrefix = redisConfig.prefix || '';
+
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: redisConfig.host,
+    options: {
+        password: redisConfig.password
+    },
+    port: redisConfig.port,
+    database: 0
+};
+
+module.exports = {
```

Comment:
`index.js` is modified in this PR: https://github.com/screwdriver-cd/queue-worker/pull/15/files#diff-168726dbe96b3ce427e7fedce31bb0bcR5

There *should* be a test to reflect that change